### DISCUSSION TITLE
not able to screech

### DISCUSSION THREAD

**Discussion by shreyas3241** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#626] [General]
when is ask google show the picture of cat it is not showing me #any think 
<img width="743" alt="Screenshot 2022-01-12 120026" src="https://user-images.githubusercontent.com/97584832/149075840-584d806c-89e6-4036-b4db-30bcb38d7f2c.png">
#

**Comment by im-coder-lg** [comment]
The Google Assistant SDK is limited in its capabilities. I think this could be resolved by enabling Personal Results. However, that'd also be limited.

**Reply by shreyas3241** [reply]
how to enable Personal Results
